src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.OperationVisitor.cs

```diff
@@ -160,15 +160,17 @@ private static void AddValue(ArrayBuilder<PlatformMethodValue> infosBuilder, str
                     using var infosBuilder = ArrayBuilder<PlatformMethodValue>.GetInstance();
                     if (PlatformMethodValue.TryDecode(method, visitedArguments, DataFlowAnalysisContext.ValueContentAnalysisResult, _osPlatformType, infosBuilder))
                     {
+                        var version = EmptyVersion;
```

Comment:
The PlatformCompatAnalyzer fails in this PR without these changes from Buyaa. IIRC, this is because this PR is updating the unit tests to use the new RC1 sdk.

Reply:
Right, some tests failed because of the sdk update, which needed a test update, plus revealed this bug. Please let me know if a separate PR is needed for this fix cc @jeffhandley

Reply:
It's OK with me to keep it combined, assuming that makes the process simpler.